docs/ruleset_functions/get_zone_supply_return_exhaust_relief_terminal_fan_power_dict.md

```diff
@@ -0,0 +1,87 @@
+
+                - Add the supply kW to this zone: `zone_total_supply_fan_power = zone_total_supply_fan_power + hvac_sys_total_supply_fan_power`    
+                - Add the return kW to this zone: `zone_total_return_fan_power = zone_total_return_fan_power + hvac_sys_total_return_fan_power` 
+                - Add the exhaust kW to this zone: `zone_total_exhaust_fan_power = zone_total_exhaust_fan_power + hvac_sys_total_exhaust_fan_power`  
+                - Add the relief kW to this zone: `zone_total_relief_fan_power = zone_total_relief_fan_power + hvac_sys_total_relief_fan_power` 
+
+                - Get list of terminal units served by the hvac system: `terminal_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["ZONE_LIST"]["Terminal_Unit_List"]`  
+                - For each terminal unit served by the HVAC system: `for terminal in terminal_list_hvac_sys_x:`  
+                    - Get the terminal unit fan power: `terminal_fan_power = get_fan_object_electric_power(RMI,terminal.fan)`  
+                    - Add to the total terminal kW associated with the zone: `zone_total_terminal_fan_power = zone_total_terminal_fan_power + terminal_fan_power`  
+
+        - Else, the hvac fan system equals Null meaning that the fan is defined at the terminal unit level like it is for a four pipe fan coil unit: `Else:`  
+            - Get list of zones served by the hvac system: `zone_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["ZONE_LIST"]`  
+            - Get list of terminal units served by the hvac system: `terminal_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["ZONE_LIST"]["Terminal_Unit_List"]`  
+            - For each terminal unit served by the HVAC system: `for terminal in terminal_list_hvac_sys_x:`  
+                - Check if the terminal unit is associated with this zone, if so then add the fan power associated with the terminal unit to the zone total supply fan power: `if terminal in zone.terminals: `  
+                    - Get the terminal unit fan power: `terminal_fan_power = get_fan_object_electric_power(RMI,terminal.fan)`  
+                    - Add to the total terminal kW for this zone: `zone_total_terminal_fan_power = zone_total_terminal_fan_power + terminal_fan_power`
+                    - Add to the zone total supply fan power since it is a supply fan defined at the terminal: `zone_total_supply_fan_power = zone_total_supply_fan_power + terminal_fan_power`  
+
+  - Add the zone as a key and the supply, return, exhaust, relief, terminal fan powers as a list of values to a dictionary: `zone_supply_return_exhaust_relief_terminal_fan_power_dict[zone.id] = [zone_total_supply_fan_power,zone_total_return_fan_power,zone_total_exhaust_fan_power,zone_total_relief_fan_power,zone_total_terminal_fan_power]`
```

Comment:
Shouldn't we also include zone.zonal_exhaust_fan power?

Reply:
Maybe change after multiple fan discussion.